react-calculator/src/components/Numbers/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+interface Props {
+  numHandler: (num: number) => void;
+}
+
+const range = (limit: number) => {
+  const result = [];
+  for (let i = 0; i < limit; i++) {
+    result.push(i);
+  }
+  return result;
+};
```

Comment:
아래와 같이 바꾸는 게 좀 더 선언적인 코드가 되겠네요!

```suggestion
const LIMIT_COUNT = 10;

const NUMBER_ARRAY = Array.from({ length: LIMIT_COUNT }, (_, index) => index);
```